src/lib.rs

```diff
@@ -108,10 +108,17 @@ pub fn basic_info_command_handler(ctx: &InfoContext, for_crash_report: bool) {
 }
 
 /// Initialize RedisModuleAPI without register as a module.
+#[cfg(feature = "use-redismodule-api")]
 pub fn init_api(ctx: &Context) {
     unsafe { crate::raw::Export_RedisModule_InitAPI(ctx.ctx) };
 }
 
+/// Initialize ValkeyModuleAPI without register as a module.
+#[cfg(not(feature = "use-redismodule-api"))]
+pub fn init_api(ctx: &Context) {
+    unsafe { crate::raw::Export_ValkeyModule_InitAPI(ctx.ctx as *mut raw::ValkeyModuleCtx) };
```

Comment:
should we add ValkeyContext in mod.rs?  

`
#[derive(Debug)]
pub struct ValkeyContext {
    pub ctx: *mut raw::ValkeyModuleCtx,
}
`

Reply:
Yes, this (handling existing structures & functions that are coupled to RedisModule points/fns) is something we should handle as part the effort here: https://github.com/valkey-io/valkeymodule-rs/issues/175

There are some ideas I had, but we can think through more to add this capability and also avoid extra code (duplications/additions)